Implement procedure to print faulty keys in YAML replacement (#64)
smash-transport/smash-vhlle-hybrid: docs/user/FAQ/index.md

    Although this might be a quick way to try out something, you should never prefer to change the base configuration files in the hybrid handler repository.
    Your changes might get in conflict with future releases of the software or simply be undone by accident using Git inside the repository.

> ### Can I collide different particles than nuclei in the IC stage?

Reviewer: ```suggestion
### Can I collide particles other than nuclei in the IC stage?
```

> You might have tried to use SMASH as initial-conditions software specifying a pion as projectile via the following configuration file and this was not accepted by the hybrid handler.

Reviewer: ```suggestion
You might have tried to use SMASH as initial-conditions software specifying a pion as the projectile via the following configuration file, but this was not accepted by the hybrid handler.
```

``` {.yaml .annotate title="Hybrid-handler configuration file"}
IC:
    Executable: "/path/to/smash"
    Software_keys:
        Modi:
            Collider:
                Projectile:
                    Particles: {211: 1} # (1)!
```

 1. :warning: This triggers an error as `Particles` is a YAML map and it contains a sub-map.
    Since `211` is a map key, which is not present in the base configuration file, this line triggers an error.

The SMASH :cloud_tornado: `IC` base configuration file is set up to collide nuclei and, therefore, both the projectile and the target are made of protons and neutrons.
These are specified **using PDG codes as YAML map keys** in the configuration file.

> Any attempt to use different particles is an attempt to try to use a new key in the configuration file and namely one that is not present in the base configuration file!

Reviewer: ```suggestion
Any attempt to use different particles is a try to use a new key in the configuration file, namely one not present in the base configuration file!
```

> If you need to collide different particles than protons and nucleons, you need your own base configuration file.

Reviewer: ```suggestion
If you need to collide particles other than protons and nucleons, you need your own base configuration file.
```

> Check out [this other FAQ](#how-can-i-add-a-software-key-to-a-configuration-file) for detailed instructions.

Reviewer: FAQ is always plural and a whole section of questions in my head, so I'd be in favor of this change:
```suggestion
Check out [this section of the FAQ](#how-can-i-add-a-software-key-to-a-configuration-file) for detailed instructions.
```
